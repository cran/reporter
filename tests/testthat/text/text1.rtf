{\rtf1\ansi\deff0 {\fonttbl {\f0 Arial;}}
\vertdoc
\paperw12240\paperh15840
\margl1440\margr1440\margt720\margb720\headery720\footery720
\sl-250\slmult0\f0\fs20
\trowd\trgaph0\trqc\cellx9360\qc Report 1.0\cell\row
\trowd\trgaph0\trqc\cellx9360\qc Simple Text Report\cell\row
\trowd\trgaph0\trqc\cellx9360\qc\sl-250\slmult0\cell\row

\li40\ri40


{\pard\fs1\sl0\par}
\li40\ri40
\trowd\trgaph0\trqc\cellx9360\ql 
Lorem ipsum dolor sit amet, 
consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et\line 
dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip\line 
ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu\line 
fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt\line 
mollit anim id est laborum.
\cell\row
{\pard\fs1\sl0\par}
\li40\ri40
{\pard\fs1\sl0  \par}}
