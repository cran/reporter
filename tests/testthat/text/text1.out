{\rtf1\ansi\deff0 {\fonttbl {\f0 Courier;}}
\vertdoc
\paperw12240\paperh15840
\margl1440\margr1440\margt720\margb720
\sl-225\slmult0\fs20
                                 Report 1.0\line
                             Simple Text Report\line
\line
Lorem ipsum dolor sit\line
 amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt\line
ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud\line
exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis\line
aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu\line
fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt\line
in culpa qui officia deserunt mollit anim id est laborum.\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line

}
